lib/archethic_web/graphql_schema/resolver.ex

```diff
@@ -122,13 +122,19 @@ defmodule ArchethicWeb.GraphQLSchema.Resolver do
     end)
   end
 
-  def get_inputs(address) do
-    case Archethic.get_transaction_inputs(address) do
-      {:ok, inputs} ->
-        {:ok, Enum.map(inputs, &TransactionInput.to_map/1)}
-
-      {:error, _} = e ->
-        e
+  def get_inputs(address, paging_offset \\ 0, limit \\ 0) do
+    inputs =
+      address
+      |> Archethic.get_transaction_inputs(paging_offset)
+      |> Enum.map(&TransactionInput.to_map/1)
+      |> Enum.sort_by(& &1.timestamp, {:desc, DateTime})
+
+    case limit do
+      0 ->
+        {:ok, inputs}
+
+      limit ->
```

Comment:
Isn't it a better practice to put the limit also in the message itself, so the nodes returns us only the data we need and not all the data ?